src/config/php.ts

```diff
@@ -0,0 +1,28 @@
+export const PHP_56 = '5.6';
+export const PHP_70 = '7.0';
+export const PHP_71 = '7.1';
+export const PHP_72 = '7.2';
+export const PHP_73 = '7.3';
+export const PHP_74 = '7.4';
+export const PHP_80 = '8.0';
+export const PHP_81 = '8.1';
+
+export const CURRENT_STABLE = PHP_74;
+
+/** NOTE: Please keep this list ordered as the ordering is used to detect the minimum supported version of a project */
+export const INSTALLABLE_VERSIONS = [
+    PHP_56,
+    PHP_70,
+    PHP_71,
+    PHP_72,
+    PHP_73,
+    PHP_74,
+    PHP_80,
+    PHP_81
+] as const;
+
+export type InstallablePhpVersionType = typeof INSTALLABLE_VERSIONS[number];
+
+export function isInstallableVersion(version: string): version is InstallablePhpVersionType {
```

Comment:
is the version provided here always in the `x.x` format, with the patch number excluded?

Reply:
❗ 
Yes, thats at least how the documentation tells the user to annotate.
We can also enforce this via some kind of JSONSchema `pattern`. 

Marking this as a TODO.